www/js/app.js

```diff
@@ -890,7 +976,7 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                 $('#searchArticles').click();
             }
         }
-        ,500);
+        ,200);
```

Comment:
Is there a reason for changing this? We actually need to slow this down for some slow browsers such as Chrome on Android...

Reply:
Actually it was asked in one of the earlier comments to make the speed faster...thats why I did this.....In my next commit, I will change this to 350 to be more optimum. You can see and suggest if it looks better

Reply:
@vasusharma7 I'm a bit confused by this still. It doesn't relate to this PR, so if it should be changed, I think it should be in a separate PR, but maybe I'm missing something.

Reply:
> Cool, it's a starting point.
> The transition works, but needs enhancement, like you said.
> 
> The goal is the user feels like the 3 sections are contiguous : clicking on the section names makes the visible zone move between these 3 sections.
> 
> ```
> -------------------
> |home |conf |about|
> |     |     |     |
> |     |     |     |
> -------------------
> ```
> 
> Currently, the transition is always from left to right : it should depend on the current section, in order to "move" to the other section in the right direction.
> I also think the transition should be a bit faster


See the last line about the transition to be made faster........Thats why I changed it to 200.

Reply:
There must be a misunderstanding here.
The number at this line is a delay between a keypress in the search field and starting a title search.
I also don't see how it is related to this PR.

What I was suggesting was to make the transition a bit faster between the sections. I suppose this could be done with the animation duration in CSS

Reply:
@mossroy @Jaifroid 
I deeply apologize for doing that.....That was my fault....I assure you not to repeat this again.
Actually I understood what you wanted me to do....While changing the animation speed, I accidentally also changed that 200 value....I have corrected and committed the corrected version....Please have a look.......I am sorry again

Reply:
No worries! Easy to mix those up, and we all make mistakes, especially when learning about new code. We're very happy to have your development help.

Reply:
@Jaifroid 
Thanks for understanding....
Now what should i do next in this PR...
Please guide further....
I dont know if it is right to ask but is it now ready for merge....
If not...what all is left now..